gadgets/python/GadgetInstrumentationStreamController.cpp

```diff
@@ -226,6 +239,14 @@ namespace Gadgetron
 	break;
       }
       break;
+      case (GADGET_MESSAGE_RECONDATA):
+     if (this->return_recondata(m0->cont()) == GADGET_FAIL);
```

Comment:
The semicolon at the end of this line is a major bug. The integration test doesn't cover this line or this entire function for that matter.